packages/experience-builder-sdk/src/core/preview/EntityStore.ts

```diff
@@ -59,7 +59,10 @@ export class EntityStore extends VisualSdkEntityStore {
     entityLink: UnresolvedLink<'Entry' | 'Asset'>,
     path: string[]
   ): string | undefined {
-    const entity = this.entitiesMap.get(entityLink.sys.id);
+    const entity =
+      entityLink.sys.linkType === 'Entry'
```

Comment:
Updated to reflect that `entitiesMap` was removed from the Visual SDK. I used `entryMap` and `assetMap` to accomplish fetching either an Entry or an Asset and used the `linkType` to determine which one to call.